src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterExtensions.cs

```diff
@@ -66,7 +62,50 @@ public static void WriteMetricsCollection(this PrometheusExporter exporter, Stre
 
                     case AggregationType.Summary:
                         {
-                            // Not supported yet.
+                            var longSummary = metric.Data as SummaryData<double>;
+                            var longValueCount = longSummary.Count;
+                            var longValueSum = longSummary.Sum;
+                            var longValueMin = longSummary.Min;
+                            var longValueMax = longSummary.Max;
+
+                            builder = builder.WithType("summary");
+
+                            foreach (var label in labels)
+                            {
+                                /* For Summary we emit one row for Sum, Count, Min, Max.
+                                Min,Max exportes as quantile 0 and 1.
+                                In future, when OT implements more aggregation algorithms,
+                                this section will need to be revisited.
+                                Sample output:
+                                MyMeasure_sum{dim1="value1"} 750 1587013352982
+                                MyMeasure_count{dim1="value1"} 5 1587013352982
+                                MyMeasure{dim1="value2",quantile="0"} 150 1587013352982
+                                MyMeasure{dim1="value2",quantile="1"} 150 1587013352982
+                                */
+                                var metricValueBuilder = builder.AddValue();
+                                metricValueBuilder.WithName(metric.MetricName + "_sum");
+                                metricValueBuilder = metricValueBuilder.WithValue(longValueSum);
+                                metricValueBuilder.WithLabel(label.Key, label.Value);
+
+                                metricValueBuilder = builder.AddValue();
+                                metricValueBuilder.WithName(metric.MetricName + "_count");
+                                metricValueBuilder = metricValueBuilder.WithValue(longValueCount);
+                                metricValueBuilder.WithLabel(label.Key, label.Value);
+
+                                metricValueBuilder = builder.AddValue();
+                                metricValueBuilder.WithName(metric.MetricName);
```

Comment:
Should min and max metric names also have a suffix like `_sum` & `_count`?

Reply:
No, min and max are exported as quantiles 0 and 1 respectively.